__tests__/app.test.js

```diff
@@ -96,6 +97,50 @@ describe('GET /api/articles/:article_id errors', () => {
     });
 })
 
+describe("GET /api/articles", () => {
+
+    test('200: responds with an array of articles ', () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then (({ body}) => { 
+            expect(Array.isArray(body.articles)).toBe(true)
+            expect(body.articles.length).toBe(13)
+            body.articles.forEach(article => {
+                expect(article).toHaveProperty('author');
+                expect(article).toHaveProperty('title');
+                expect(article).toHaveProperty('article_id');
+                expect(article).toHaveProperty('topic');
+                expect(article).toHaveProperty('created_at');
+                expect(article).toHaveProperty('votes');
+                expect(article).toHaveProperty('article_img_url');
+                expect(article).toHaveProperty('comment_count');  
```

Comment:
You're making sure that the properties exist, but you're not verifying anything about the value itself - these could all have a value of null for example and still pass this test.

To test the properties **and values** of the entire response object, you might find it helpful to use a flexible matcher like [.toMatchObject](https://jestjs.io/docs/expect#expectobjectcontainingobject) or [objectContaining](https://jestjs.io/docs/expect#expectobjectcontainingobject) combined with `expect.any(*data type*)` in order to check this response object - have a look into this! (also, .toMatchObject ignores extra properties which leaves room for expansion later on)